docs/en/latest/api/api.md

```diff
@@ -25,6 +25,44 @@ Manager API directly operates ETCD and provides data management for Apache APISI
 
 **License:** [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0)
 
+### /apisix/admin/migrate/export
+
+#### GET
+
+##### Summary:
+
+Export a config file for migrate.
+
+##### Parameters
+
+None.
+
+##### Responses
+
+A file for download.
+
+### /apisix/admin/migrate/import
+
+##### Summary:
+
+Import the config file for restore config.
+
+#### POST
+
+##### Parameters (FORM)
```

Comment:
@fregie As current APIs provided by manager-api are all of RESTful-JSON, instead of FORM. Please also see OpenAPI export/import  APIs for example. So how about we also add support for that format?

Reply:
Using FORM is for upload a file.We can also aupport json or openAPI.Maybe in another PR?

Reply:
> Using FORM is for upload a file.We can also aupport json or openAPI.Maybe in another PR?

@fregie Sure and thanks.